fragments/shared/users/python3_check_root_locked.ks

```diff
@@ -0,0 +1,27 @@
+# check root account is locked
+%post --interpreter=/usr/bin/python3
+
+import sys
+import crypt
```

Comment:
do we need to import crypt here?

Reply:
Actually, good point, it got carried over from the original variant that also validated the password is correct, which unfortunately can't really be deduplicated via fragments due to needing to know the password.

Might be something to think about for the future - basically some argument/input passing for fragments, which would make more cases de-duplicable when the same check code needs to work on different kinds of input. 

Two ways com to my mind:
- some standardized files being dropped in place by previous snippet
- support for substitution inside of fragments in some way (things might become meta if we are not careful with this ;-) )